AcadeMeData/tests.py

```diff
@@ -1,3 +1,34 @@
-# from django.test import TestCase
-
-# Create your tests here.
+from django.test import TestCase
+from AcadeMeData.models import Users
+
+
+class TestUserModel(TestCase):

```

Comment:
It is agreed that we are using django-pytest. no test cases
Please review last class

Reply:
RESOLVED